src/hatti/ona/forms.cljs

```diff
@@ -219,7 +219,10 @@
                                (filter #(contains? names (field-key %)))
                                (map #(str "☑ " (get-label % language) " "))
                                string/join))
-    (time-based? field) (-> answer js/moment (.format "ll"))
+    (time-based? field) (if (or (js/isNaN (new js/Date answer))
```

Comment:
This seems reusable and appropriate for a helper. If it's in a helper, we could move it to the lhs of the `cond`, i.e. `(and (time-based? field) (valid-date? answer))`

Also, the constructor `(new js/Date answer)` is repeated, we could avoid that w/a let at worst